src/instruments/src/EFB/Localization/en.json

```diff
@@ -487,8 +487,8 @@
       "ConesEnabled": "Safety Cones",
       "DefaultBarometerUnit": "Default Barometer Unit",
       "DynamicRegistrationDecal": "Dynamic Registration Decal",
-      "EnabledMcduServerConnectionAutoDeactivatesAfter5MinutesIfNoSuccessfulConnection": "Enable MCDU Server Connection (Auto deactivates after 5 minutes if no successful connection)",
-      "ExternalMcduServerPort": "External MCDU Server Port",
+      "EnabledSimbridgeConnectionAutoDeactivatesAfter5MinutesIfNoSuccessfulConnection": "Enable Simbridge Connection (Auto deactivates after 5 minutes if no successful connection)",
```

Comment:
For these localisation strings, you may have to coordinate with the localisation team to have these fixed, or just revert to using the older keys, which may be easier given that localazy is not exactly great at handling changing keys